CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ ___Note:__ Yet to be released changes appear here._
 ### BPMN
 
 * `FEAT`: support converging inclusive gateway ([#3613](https://github.com/camunda/camunda-modeler/issues/3613))
+* `FIX`: improve `no-loop` performance ([bpmnlint-plugin-camunda-compat#165](https://github.com/camunda/bpmnlint-plugin-camunda-compat/pull/165))
```

Comment:
```suggestion
* `FIX`: improve `no-loop` performance ([camunda/bpmnlint-plugin-camunda-compat#165](https://github.com/camunda/bpmnlint-plugin-camunda-compat/pull/165))
```

Reply:
We prefix the org usually, cf. existing changelog entries.

Reply:
I see! I thought we only prefix the org if it is different than the current one (same as github does with auto-shortening links)

Reply:
Let's discuss, but be pragmatic. 

For me I'd like to be able to just copy things over from one project to another, without thinking too much (i.e. if I shall adopt the org prefix).

Reply:
I'm fine with this as well, was just not aware 😄